client/components/common/tce-core/Assessment/index.vue

```diff
@@ -113,7 +122,9 @@ export default {
       if (!this.isEditing) return;
       this.errors = [];
       this.validate().then(() => {
-        this.$emit('save', cloneDeep(this.editedElement.data));
+        const elementData = cloneDeep(this.editedElement.data);
+        if (!this.isGraded) delete elementData.correct;
```

Comment:
Can we hit this scenario if we init data based on type and we disable correct input?